src/COMDIRAC/__init__.py

```diff
@@ -12,4 +12,4 @@
 
 
 def extension_metadata():
-    return {"priority": 10}
+    return {"priority": 11}
```

Comment:
Does this need to be an extension at all? It doesn't override anything does it?

Reply:
To be clear, I mean it doesn't need to be a "DIRAC extension" and can just be a standard python package that depends on `DIRAC`.

Reply:
this package also has an `Interfaces/scripts` directory for ex.

So this function is not needed? can i delete it?

Reply:
Ah it's using the DIRAC argument parser. In that case it's probably easier to leave it as is.